src/Fantomas.Client/FantomasToolLocator.fs

```diff
@@ -48,7 +49,18 @@ let private readOutputStreamAsLines (outputStream: StreamReader) : string list =
 
     readLines outputStream id
 
-let private runToolListCmd (Folder workingDir) (globalFlag: bool) =
+let private startProcess (ps: ProcessStartInfo) : Result<Process, ProcessStartError> =
+    try
+        Ok(Process.Start ps)
+    with
+    | :? Win32Exception as win32ex ->
+        let pathEnv =
+            Environment.GetEnvironmentVariable "PATH"
+
+        Error(ProcessStartError.ExecutableFileNotFound(ps.FileName, ps.Arguments, pathEnv, win32ex.Message))
```

Comment:
the working directory would be useful information for these errors as well, that would be the most complete representation of the execution environment.

Reply:
Good catch